algorithms/kernel/k_nearest_neighbors/bf_knn_classification_predict_fpt_ucapi.cpp

```diff
@@ -0,0 +1,46 @@
+/* file: bf_knn_classification_predict_fpt_ucapi.cpp */
+/*******************************************************************************
+* Copyright 2014-2019 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+/*
+//++
+//  Implementation of prediction stage of K-Nearest Neighbors algorithm.
```

Comment:
```suggestion
//  Implementation of prediction stage of Brute-force K-Nearest Neighbors algorithm.
```
Do we need to mark up that it is a BF-KNN?